portfolio/templates/index.html

```diff
@@ -18,6 +18,13 @@
             line-height: 1.2
         }
     </style>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=TAG_ID"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'G-JCX3Z8JZJC');
```

Comment:
Is it possible to reference the ID from the yaml file?

Reply:
good call, I templated it in everywhere via Jinja